packages/apputils/src/breadcrumbs.tsx

```diff
@@ -32,9 +31,9 @@ export class Breadcrumbs extends React.PureComponent<IBreadcrumbsProps> {
         {items.map((item) =>
           item.onClick ? (
             <BreadcrumbItem key={item.text}>
-              <Button appearance="lightweight" onClick={item.onClick}>
+              <a style={{ color: "var(--jp-brand-color1)" }} onClick={item.onClick}>
```

Comment:
Just one organization comment; you should prefer setting a `className` and placing the style in `apputils/style/base.css`.